backend/src/services/auth.service.ts

```diff
@@ -36,8 +33,28 @@ export async function authUser(code: string): Promise<string> {
       apidata: userFromApi,
       access_token: tokenResponse.data.access_token,
       refresh_token: tokenResponse?.data?.refresh_token ?? null,
+      vacdm: {},
     };
 
+    // Auth user from VACC Auth URL
+    if (config().vaccAuthUrl !== undefined && config().vaccAuthToken !== undefined) {
+      try {
+        const vaccAuthResponse = await axios.get(config().vaccAuthUrl, {
+          headers: {
+            Authorization: 'Bearer ' + config().vaccAuthToken,
+          },
+          params: {
+            vatsimid: userFromApi.cid,
+          },
+        });
+
+        if (vaccAuthResponse.status == 200) updateOps.vacdm.atc = true;
+        else updateOps.vacdm.atc = false;
+      } catch (err) {
+        updateOps.vacdm.atc = false;
+      }
+    }
+
```

Comment:
This should probably be moved to another service, as it might be necessary to implement other similar functionality next to this. That would allow some abstraction to happen.

In a similar fashion, it might be necessary to add an environment variable allowing one to select the vacc auth mode. That way, you could have the "austrian functionality" but other vaccs can have similar functionality without needing to support your api.

Reply:
Of course!

Regarding the service: I think it is still something which should happen in the auth service, since it is a form of authentication. 
I would move the function into another function where you can switch between different modes, based on an environment variable, as you said!